WalletWasabi.Backend/Controllers/BlockchainController.cs

```diff
@@ -210,6 +210,20 @@ private async Task<Transaction[]> FetchTransactionsAsync(uint256[] txIds, Cancel
 				{
 					txIdsRetrieve[kvp.Key].TrySetResult(kvp.Value);
 				}
+
+				var stillMissing = txIdsRetrieve.Where(x => !rpcBatch.ContainsKey(x.Key)).ToList();
```

Comment:
Let's say that we fetch `A` and `B`. `A` is in the mempool but not `B`, neither are in `Cache` (it doesn't matter whether `A` is in `Cache` or not)

`txIdsRetrieve` contains  `{{A, A_tcs}, {B, B_tcs}}`.
`rpcBatch` will return `{{A, A_raw}}` but `B` is missing.
`TrySetResult` will be called on `A_tcs`, but not on `B_tcs`

Therefore currently, on Line `Transaction tx = await txsCompletionSources[i].Task.ConfigureAwait(false);`, when we are on `A_tcs` it will immediately return. But on `B_tcs`? We are waiting, but waiting for what? Nothing is happening in the background. So we are dumbly waiting for the cancellation token to kick in.

So this PR sets an exception on the tcs where no results where set, and instantly throw as one tx was not available. We can have different concepts, for example return the txs that were successfully caught without throwing, but in all cases we shouldn't `await B_tcs`.